### PR TITLE
Carousel form - buttons for switching slides/metrics

### PR DESCRIPTION
**Pull request by brandi-jones**
Customized the methods of switching between metric slides on the carousel. The next/previous functions that control activeIndex state are passed down as props to each metric slide, then passed again as props down to each metric question slide where the arrow buttons are located. onClick of these arrow buttons call the next and previous functions accordingly

### CODE REVIEW

**Review by alvillaraza — APPROVED** [review]
This looks great! It doesn't interfere with current code, and it looks clean. I checked AWS, and it works fine.

File: src/components/DailyRetro/Form/Form.js [hunk 1]
```diff
@@ -56,6 +39,22 @@ function Form(props) {
     setActiveIndex(newIndex);
   };
 
+  //in documentation, items are in an object form? may need to change if we have any trouble
+  const items = [
```
Comment (ryan-hamblin) [inline src/components/DailyRetro/Form/Form.js:43]: Formatting seems a bit off here. Consider using a linter/prettier for auto-formatting in workflow. 

Please indent the objects in this array.

Comment (ryan-hamblin) [reply src/components/DailyRetro/Form/Form.js:43]: consider indenting lines 44 -> 56

File: src/components/DailyRetro/Form/Form.js [hunk 2]
```diff
@@ -5,26 +5,9 @@ import SleepSlide from "../CarouselForm/slides/Sleep/SleepSlide.js";
 import BreaksSlide from "../CarouselForm/slides/Breaks/BreaksSlide.js";
 import {
   Carousel,
-  CarouselItem,
-  CarouselControl,
-  CarouselIndicators,
+  CarouselItem
 } from "reactstrap";
 
-//in documentation, items are in an object form? may need to change if we have any trouble
```
Comment (ryan-hamblin) [inline src/components/DailyRetro/Form/Form.js:13]: Love you that reformatted and removed so much code! It really seems to have cleaned up the code base!

**Review by ryan-hamblin — COMMENTED** [review]
The work here is very good. Just a few minor tweaks to consider in places that I've commented inline.

**Pull request merged**